mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -193,6 +193,7 @@ export class ExperimentListView extends Component {
                       onClick={this.handleDeleteExperiment}
                       data-experimentid={experiment_id}
                       data-experimentname={name}
+                      style={{ marginRight: 10 }}
```

Comment:
Change to fix https://github.com/mlflow/mlflow/pull/5577#pullrequestreview-931321957